### ISSUE TITLE
Release Cycles lists web site that redirects to malicious scam site

### ISSUE THREAD

**Issue by ProactiveServices** [leereilly/games#154]
The web site listed on https://github.com/RothschildGames/release-cycles redirects to a malware fake security site. I assume the domain/web hosting expired and was snapped up by a scam outfit. You may wish to remove this entry from your list so other users don't fall foul.

**Comment by vicenterusso** [comment]
I disagree. The full source are still there. And, if your talking about "http://release-cycles.com", there is no malware or whatever you think it is.. just a standard domain without content owned by http://www.afternic.com

**Comment by simplypeachy** [comment]
I assume it's on some sort of round-robin redirect. The first time I visited:
(warning: contains scam site web sites)
2015-09-27 18:33:29.619 000016d4 Request: http://release-cycles.com/
2015-09-27 18:33:30.181 000016d4 Request: http://release-cycles.com/?ga=pQ2sXDwp3XYJJ1jWgqpPkAoTTNj3gHqMTgAXgsQfsINtQToYhvr5qi%2BZgUDYqigqHxXJySfQbywt23PTFY9uhg%3D%3D&gerf=E8kIPkVa393d1uuBe2CZceYMb2AYlEjrZ59fPgYXTxw%3D&guro=APvaVJ%2BQA6sOYdHCSRXR1BKhYdE3unWnZEXxFUmXwGGAX64Ulin3vJyVwD3mtLsP&
2015-09-27 18:33:32.352 000016d4 Request: http://bestfwdservice.com/adclk?&gm=koCcREJkqhfkuUNHri81gyeE2rvKxyXGUUUBjpuZOUGnS%2FV7cQar%2BI0TkcVhW93Q8TMVOROD%2FGjUohQt2dkYmADrHogqokra9avMF5OWpW6sxiEyHpnyBFgDqiTENXGTeaIypP%2B6MabkBFf8kIuhbbquMatLPLV8EGEqB515UUIdpEQpDDv5AEDlbrD1hzxBieqRgbgSKixZkSLV2YGORcciuZge4aH%2BAB18q7sbIytfgKk5%2BEofoS3cc9jmDg6in12TnNU3s93izWPkTI1cCpyHLvU7CrRriLuTX%2B%2BOSChzPbSVdnYGzQ3Gm0i2iQfDVTJwvunS5rq65BjwU3vLvizO884XZBaDJVR3j358rG39TC5GePa7O%2FZsidxHe0BB&gc=11071909208101111659229&gi=JGKBhSjuBIupwSb9qUFDZssXG2GvK5ATUksvPXlIUvy77Cc%2FwebCE1A9I1ll9X28xvTehCULJzTtMmHtStfyIVePDkL0YTbtvjf2x%2B8me%2Fxy2yBPP0pJIImThMPsUJllVAGLkHvyGr0ImDFpNmMxcFCpAFSm%2BdS34OsRyi9qTKbXZZQYg3xz0gi7vdrFPVipXEAHKu9WcWFT942Ioxplaq2%2Bxrl0wZK7sg5ApXjb32eMoN7qotJ0OCqzl%2F9ZeSGemO6vdljj2HCdK6uS9T%2FX6bH1OBhgLwhyOuPgEcbTkhNQSTAVwzaI0VZGaJ%2B529XgBH90svQU99M7wvC517fFJkOEdjb0EhCQTVouWIv7s4VrWlWK8bzRX1OUGC0luf%2FUVkRvKeCJzX%2B4gptoxIh3LibN7PEY47E6ejE31F1i%2FQs%3D&jccheck=1&zchdrdr=1
2015-09-27 18:33:33.867 00001154 Request: http://bestfwdservice.com/rg-clkrdr.php?_t=zro&_d=Nfaf19fcJqJaf9.JvV&_p=t%20nGGS%20OE&_pr=GbbZHZb&_v=zbTztbtGbAzbzzzzZHt&_rdfu=X55g9%3ADDgG.4B5N1Y.JvVD5N%3FW4P4%2FHG1QtbE1H1EfEzFAETGFH%2FbT1H14b4FzftZHAQ.BN&_bku=X55g%3ADDQi49fNmWJf.JvVD%3F%26s1Pz+2z1RJ1TxflOtGk%2FsvATtIC%25G_+-sQmu%25Gh1iRlNC-FHT%2FiWS47lSw4t-OFnK%3D2_7V0vtN%25G_m0THLVBSjyz03kVxGwmfqRs%25G_%25G_e%25Gh3tG4FHEB5urt6RIYqT70I9JF1KW_xMsZK%25G_aMnsEKO2v8bjk46amYZHi%25FI%25FI
2015-09-27 18:33:34.216 00001a78 Request: http://p2.dntrax.com:443/
2015-09-27 18:33:34.763 00000c80 Request: http://p2.dntrax.com:443/
2015-09-27 18:33:34.846 00001070 Crunch: Blocked: http://www.dntx.com:443
2015-09-27 18:33:35.142 000016d4 Crunch: Blocked: http://http://opitmalsurveys.com/se/dnx4/index.php
2015-09-27 18:33:35.301 00001070 Crunch: Blocked: http://www.dntx.com:443
2015-09-27 18:33:35.312 00001070 Crunch: Blocked: http://www.dntx.com:443
2015-09-27 18:33:47.742 000016d4 Request: http://opitmalsurveys.com/se/dnx4/index.php
2015-09-27 18:33:48.251 000016d4 Request: http://7drcu.trackvoluum.com/ac86e44b-dd01-4d48-87ef-2f48cb9096af?sourceid=[sourceid]&match=[match]&carrier=[carrier]&mob_pf=[mob_pf]&country=[country]&cpc=[cpc]
2015-09-27 18:33:48.411 000016d4 Request: http://support4you.assist-you.xyz/msax4/index.html?ip=[redacted]&city=[redacted]&num=02038-155194&voluumdata=vid..00000000-4dbf-41c6-8000-000000000000__vpid..3025e800-6539-11e5-83ff-0a73ea932b69__caid..ac86e44b-dd01-4d48-87ef-2f48cb9096af__rt..H__lid..f23b63cc-8e31-492a-821a-925ad24ec5cd__oid1..26f1af36-6948-4e3c-b5b2-b6e069906e9e__var1..%5Bsourceid%5D__var2..%5Bmatch%5D__var3..%5Bcarrier%5D__var4..%5Bmob%5C_%5Cpf%5D__var5..%5Bcountry%5D__rd..opitmalsurveys%5C.%5Ccom__aid..__sid..&sourceid=[sourceid]&match=[match]&carrier=[carrier]&mob_pf=[mob_pf]&country=[country]&cpc=[cpc]

**Comment by leereilly** [comment]
This repo is now being archived, so I’ll be closing this out. I’m really grateful you took the time to share and be part of this project <3
